css/style.css

```diff
@@ -1,9 +1,14 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
```

Comment:
I would remove development comments like this as they will be visible in the source code.